package.json

```diff
@@ -19,12 +19,13 @@
     "test-cypress": "cypress open",
     "test-cypress-build": "cypress run --spec ./cypress/features/build/build*.feature --browser chrome --reporter spec",
     "test-cypress-accessibility": "cypress run --spec ./cypress/features/accessibility/accessibility.feature --browser chrome --reporter spec",
-    "test-cypress-regression": "npm run cypress-group-regression-common; npm run cypress-group-regression-experimental; npm run cypress-group-regression-deprecated; npm run cypress-group-regression-validations; npm run cypress-group-regression-themes; npm run generate-cypress-allure-report",
-    "cypress-group-regression-common": "cypress run --spec './cypress/features/regression/*.feature' --reporter mocha-allure-reporter --browser chrome",
-    "cypress-group-regression-experimental": "cypress run --spec './cypress/features/regression/experimental/*.feature' --reporter mocha-allure-reporter --browser chrome",
-    "cypress-group-regression-deprecated": "cypress run --spec './cypress/features/regression/deprecated/*.feature' --reporter mocha-allure-reporter --browser chrome",
-    "cypress-group-regression-validations": "cypress run --spec './cypress/features/regression/validations/*.feature' --reporter mocha-allure-reporter --browser chrome",
-    "cypress-group-regression-themes": "cypress run --spec './cypress/features/themes/themes.feature' --reporter mocha-allure-reporter --browser chrome",
+    "test-cypress-regression": "npm run test-cypress-accessibility; npm run cypress-group-regression-validations; npm run cypress-group-regression-themes; npm run cypress-group-regression-common; npm run cypress-group-regression-experimental; npm run cypress-group-regression-deprecated",
```

Comment:
`test-cypress-regression` and `cypress-group-regression-common` both run the same tests. 

I prefer the `cypress-group-regression-common` but could we rename it to `cypress-group-regression-all`and remove `test-cypress-regression`. I prefer it because of the glob matching all files.

Reply:
I've divided the scripts that way to have possibility run npm script from Terminal